mern/client/src/screens/withRouter 2.js

```diff
@@ -0,0 +1,16 @@
+import { useNavigate } from 'react-router-dom';
```

Comment:
Why do we have this file again?? We originally had withRouter.js right? Nothing looks different to me... 🤔